tunnel.go

```diff
@@ -103,12 +83,31 @@ func (t *Tunnel) ForwardPort() error {
 
 	select {
 	case err = <-errChan:
-		return microerror.Mask(err)
+		return nil, microerror.Mask(err)
 	case <-pf.Ready:
-		return nil
+		return tunnel, nil
 	}
 }
 
+type TunnelConfig struct {
+	Remote    int
+	Namespace string
+	PodName   string
+}
+
+type Tunnel struct {
+	TunnelConfig
+	Local int
+
+	stopChan chan struct{}
+}
+
+// Close disconnects a tunnel connection.
+func (t *Tunnel) Close() error {
```

Comment:
Can this return something != nil?

Reply:
No, but it fulfils io.Closer interface now.

Reply:
See https://golang.org/pkg/io/#Closer Close return an error. I think it's always good to conform to basic interfaces. WDYT?

Reply:
Ok sounds good :+1: maybe a comment would make things more clear.

Reply:
Added some godoc.